src/sync.cpp

```diff
@@ -78,12 +78,12 @@ namespace librealsense
         :matcher({stream})
     {
         _streams_type = {stream_type};
-        _name = "I " + std::string(rs2_stream_to_string(stream_type));
+        _name = std::string(rs2_stream_to_string(stream_type));
```

Comment:
why remove the 'I'?

Reply:
Wasn't useful -- anything without `()` was I